### PR TITLE
[WIP][Zeta E2E] enable zeta e2e

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

**Comment by hailin0** [comment]
check ci error

File: docs/en/connector-v2/source/LocalFile.md [hunk 1]
```diff
@@ -200,6 +200,9 @@ connector will generate data as the following:
 
 The schema information of upstream data.
 
+<<<<<<< HEAD
```
Comment (TyrantLucifer) [inline docs/en/connector-v2/source/LocalFile.md]: revert

File: docs/en/connector-v2/source/LocalFile.md [hunk 2]
```diff
@@ -200,6 +200,9 @@ connector will generate data as the following:
...
+============
```
Comment (TyrantLucifer) [inline docs/en/connector-v2/source/LocalFile.md]: revert

File: docs/en/connector-v2/source/LocalFile.md [hunk 3]
```diff
@@ -214,7 +217,9 @@ The file type supported column projection as the following shown:
 
 **Tips: If the user wants to use this feature when reading `text` `json` `csv` files, the schema option must be configured**
 
-### common options
+>>>>>>> apache/dev
```
Comment (TyrantLucifer) [inline docs/en/connector-v2/source/LocalFile.md]: revert

File: seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java [hunk 4]
```diff
@@ -70,6 +72,7 @@ public void close() throws IOException {
 
     @Override
     public void addSplitsBack(List<FakeSourceSplit> splits, int subtaskId) {
+        log.debug("Fake source add splits back========{}, subtaskId:{}", splits, subtaskId);
```
Comment (TyrantLucifer) [inline seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java]:
```suggestion
        log.debug("Fake source add splits back {}, subtaskId:{}", splits, subtaskId);
```

File: seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java [hunk 5]
```diff
@@ -88,7 +91,11 @@ public void registerReader(int subtaskId) {
 
     @Override
     public FakeSourceState snapshotState(long checkpointId) throws Exception {
-        return new FakeSourceState(assignedSplits);
+        log.debug("get lock, begin snapshot fakesource split enumerator...");
```
Comment (TyrantLucifer) [inline seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java]:
```suggestion
        log.debug("Get lock, begin snapshot fakesource split enumerator...");
```

File: seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java [hunk 6]
```diff
@@ -88,7 +91,11 @@ public void registerReader(int subtaskId) {
...
+        synchronized (lock) {
+            log.debug("begin snapshot fakesource split enumerator...");
```
Comment (TyrantLucifer) [inline seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeSourceSplitEnumerator.java]:
```suggestion
            log.debug("Begin snapshot fakesource split enumerator...");
```

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java [hunk 7]
```diff
@@ -810,56 +830,77 @@ public void testStreamJobRestoreInAllNodeDown()
             node1.shutdown();
             node2.shutdown();
 
+            System.out.println(
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java]:
```suggestion
            log.info(
```

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java [hunk 8]
```diff
@@ -810,56 +830,77 @@ public void testStreamJobRestoreInAllNodeDown()
...
             HazelcastInstanceImpl restoreFinalNode = node1;
             Awaitility.await()
-                    .atMost(10000, TimeUnit.MILLISECONDS)
+                    .atMost(60000, TimeUnit.MILLISECONDS)
                     .untilAsserted(
                             () ->
                                     Assertions.assertEquals(
                                             2, restoreFinalNode.getCluster().getMembers().size()));
 
+            System.out.println(
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java]:
```suggestion
            log.info(
```

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java [hunk 9]
```diff
@@ -810,56 +830,77 @@ public void testStreamJobRestoreInAllNodeDown()
...
+                                        JobStatus.RUNNING.equals(jobStatus)
                                                 && testRowNumber * testParallelism
                                                         == FileUtils.getFileLineNumberFromDir(
                                                                 testResources.getLeft()));
                             });
 
             // sleep 10s and expect the job don't write more rows.
             Thread.sleep(10000);
-            clientJobProxy.cancelJob();
+            System.out.println(
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java]:
```suggestion
            log.info(
```

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java [hunk 10]
```diff
@@ -810,56 +830,77 @@ public void testStreamJobRestoreInAllNodeDown()
...
-                                                            objectCompletableFuture.get())));
-
+                                                            waitForJobCompleteFuture.get())));
             // prove that the task was restarted
             Long fileLineNumberFromDir =
                     FileUtils.getFileLineNumberFromDir(testResources.getLeft());
             Assertions.assertEquals(testRowNumber * testParallelism, fileLineNumberFromDir);
 
         } finally {
+            System.out.println(
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterFaultToleranceIT.java]:
```suggestion
            log.info(
```

File: docs/en/connector-v2/source/LocalFile.md (hunk 3 above)
Comment (EricJoy2048) [reply docs/en/connector-v2/source/LocalFile.md]: done

File: docs/en/connector-v2/source/LocalFile.md (hunk 2 above)
Comment (EricJoy2048) [reply docs/en/connector-v2/source/LocalFile.md]: done

File: docs/en/connector-v2/source/LocalFile.md (hunk 1 above)
Comment (EricJoy2048) [reply docs/en/connector-v2/source/LocalFile.md]: done

**Pull request closed without merging**